spec.html

```diff
@@ -936,6 +989,30 @@ <h1>The AsyncContext Object</h1>
 
     <emu-clause id="sec-asynccontext-abstract-operations">
       <h1>AsyncContext Abstract Operations</h1>
+
+      <emu-clause id="sec-hostgettoplevelasynccontextmapping" type="abstract operation">
+        <h1>
+          HostGetTopLevelAsyncContextMapping (
+            _module_: a Module Record or *null*
+          ): a List of Async Context Mapping Records</h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>It allows hosts to provide the top level async context mapping.</dd>
+        </dl>
+        <p>An implementation of HostGetTopLevelAsyncContextMapping must conform to the following requirements:</p>
+        <ul>
+          <li>It must return a List of Async Context Mapping Records.</li>
```

Comment:
I still think this requirement should be deleted, since it's completely redundant with the signature.  Looking through the other host-defined ops, I see a few with non-unused returns (e.g. [HostHasSourceTextAvailable](https://262.ecma-international.org/15.0/#sec-hosthassourcetextavailable) and [HostResizeArrayBuffer](https://262.ecma-international.org/15.0/#sec-hostresizearraybuffer)) and none of them have this sort of restatement of the signature as an enumerated requirement.